spec/packet_logs/packet_log_reader_spec.rb

```diff
@@ -125,7 +125,7 @@ module Cosmos
           file.write [pkt.buffer.length].pack('N')
           file.write pkt.buffer
         end
-        @plr.open(filename)
+        expect(@plr.open(filename)).to eql [false, nil]
```

Comment:
There are no specs where we check the second parameter to not be anything but nil.